CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ The [Open Source Guides](https://opensource.guide/) website has a collection of
 There are many ways to contribute to Svelte, and many of them do not involve writing any code. Here's a few ideas to get started:
 
 - Simply start using Svelte. Go through the [Getting Started](https://svelte.dev/blog/the-easiest-way-to-get-started) guide. Does everything work as expected? If not, we're always looking for improvements. Let us know by [opening an issue](#reporting-new-issues).
-- Look through the [open issues](https://github.com/sveltejs/svelte/issues). Provide workarounds, ask for clarification, or suggest labels. Help [triage issues](#triaging-issues-and-pull-requests).
+- Look through the [open issues](https://github.com/sveltejs/svelte/issues). A good starting point is issues tagged [good first issue](https://github.com/sveltejs/svelte/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Provide workarounds, ask for clarification, or suggest labels. Help [triage issues](#triaging-issues-and-pull-requests).
```

Comment:
nit: "A good starting point is issues..." sounds a bit off. How about "A good starting point would be issues..."

Reply:
Labels have been reworked since this PR opened. Link currently points to nothing.

Looks like we want [this](https://github.com/sveltejs/svelte/issues?q=is%3Aopen+is%3Aissue+label%3A%22triage%3A+good+first+issue%22) instead.

Reply:
looks like something switched back because current link in docs works for me and Luke's doesn't :smile:

Reply:
```suggestion
- Look through the [open issues](https://github.com/sveltejs/svelte/issues). A good starting point would be issues tagged [good first issue](https://github.com/sveltejs/svelte/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). Provide workarounds, ask for clarification, or suggest labels. Help [triage issues](#triaging-issues-and-pull-requests).
```